server/services/IndexService.ts

```diff
@@ -77,12 +77,12 @@ export default class IndexService {
         callWithRequest("cat.recovery", {
           format: "json",
           detailed: true,
-        }),
+        }).catch(() => []),
```

Comment:
should we only fix for missing permission case (http status code 403) or all exception case

Reply:
Actually no matter what error cat.recovery API gives we should wrap and gives the raw response the cat.indexes API gives.